apps/prairielearn/package.json

```diff
@@ -21,6 +21,7 @@
     "@aws-sdk/client-sqs": "^3.631.0",
     "@aws-sdk/credential-providers": "^3.631.0",
     "@aws-sdk/lib-storage": "^3.631.0",
+    "@faker-js/faker": "^8.4.1",
```

Comment:
This is a chunky old package: https://bundlephobia.com/package/@faker-js/faker@8.4.1. Per npm, it's almost 11MB unpacked.

Can we get away with a relatively small hardcoded list of first/last names?

Reply:
Alternatives:
* https://www.npmjs.com/package/casual (very small, but 5 years old)
* https://www.npmjs.com/package/minifaker (2 years old)
* https://www.npmjs.com/package/@ngneat/falso

Reply:
I guess I don't see why we don't just use a hardcoded list like the sproc did? Those names are ~15kB, as opposed to at least 400kB for the smallest of those three.

Reply:
Done in 9ab9db3f65dcac5cdf21039968940db75921c16c. I copied the names from faker.js, adding up to a 44KB library, I'm happy to trim that further if you'd like.